pom.xml

```diff
@@ -46,17 +53,95 @@
         </dependency>
         <dependency>
             <groupId>io.opentelemetry</groupId>
-            <artifactId>opentelemetry-sdk-common</artifactId>
+            <artifactId>opentelemetry-sdk</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.opentelemetry</groupId>
+            <artifactId>opentelemetry-semconv</artifactId>
+            <version>1.2.0-alpha</version>
```

Comment:
I suggest to look at the idea of using a Maven pom.xml property `opentelemetry.version` to make it easier to bumpthe  version of the OpenTelemetry SDK. See https://github.com/jenkinsci/opentelemetry-plugin/blob/0.14-beta/pom.xml#L25

Reply:
Thank you! I fixed it.